### PR TITLE
Added ability to pickup objects and added dash move

### PR DESCRIPTION
**Pull request by mbajaman**

### CODE REVIEW

File: Assets/Scripts/GameManager.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using System.Collections;
```
Comment (AlexisCM) [inline Assets/Scripts/GameManager.cs:1]: Remove unnecessary `using` statements on Lines 1 and 2

File: Assets/Scripts/GameManager.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    void Start() {
+        _keyCollected = false;
+        _keyObject = GameObject.Find("Glitch(Key)");
+        _keyUI = GameObject.Find("UIKey");
+        
+    }
+
+    void Update() {
+        if (!_keyObject.activeSelf)
```
Comment (AlexisCM) [inline Assets/Scripts/GameManager.cs:20]: Use curly braces even when optional

File: Assets/Scripts/GameManager.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            _keyCollected = true;
+
+        if(_keyCollected) {
```
Comment (AlexisCM) [inline Assets/Scripts/GameManager.cs:23]: Space between `if` operator and opening parenthesis

File: Assets/Scripts/PlayerController.cs [hunk 4]
```diff
@@ -3,6 +3,8 @@
 
 [RequireComponent(typeof(CharacterController), typeof(PlayerInput))]
 public class PlayerController : MonoBehaviour {
+    public Vector3 move;
```
Comment (AlexisCM) [inline Assets/Scripts/PlayerController.cs:6]: Public member variables are PascalCase, i.e. `Move`

File: Assets/Scripts/PlayerDash.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
+using System.Collections;
```
Comment (AlexisCM) [inline Assets/Scripts/PlayerDash.cs:1]: Remove unnecessary `using` statements

File: Assets/Scripts/PlayerDash.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerDash : MonoBehaviour
+{
+    public float dashSpeed;
```
Comment (AlexisCM) [inline Assets/Scripts/PlayerDash.cs:7]: Public member variables are PascalCase

File: Assets/Scripts/ObjectsToCollect.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ObjectsToCollect : MonoBehaviour
+{
+    private void OnTriggerEnter(Collider player)
+    {
+        if (player.gameObject.tag == "Blob")
```
Comment (AlexisCM) [inline Assets/Scripts/ObjectsToCollect.cs:9]: Use curly braces even when optional

File: Assets/Scripts/GameManager.cs (hunk 3 above)
Comment (mbajaman) [reply Assets/Scripts/GameManager.cs]: Fixed spacing in if statement

**Review by Charles-Huang-Y4 — APPROVED** [review]

**Pull request merged**